GameDev/week6/homework/issues/audioHomework.md

```diff
@@ -0,0 +1,5 @@
+Now, what I’d like for you to do this week is to find some sound assets of your own, either using the websites I’ve shown you, some other source, or even your own recording. Even though we’re not working on this project commercially, be aware of the audio’s licensing as you’re searching. Ideally, get one background music track, and two sound effects for jumping and crashing. Add them to the project folder and integrate them into this game!
+
+For further mastery of audio:
```

Comment:
I like the bulleted formatting! However, you have to put a space between "-" and your content in order for it to show as bullet points. Try using the preview button shown below to preview what your file looks like in rich format!

![image](https://user-images.githubusercontent.com/69332964/104095257-e577b680-5263-11eb-8194-7ba973a0703d.png)